E/Enzyme/build_tarballs.jl

```diff
@@ -38,13 +38,14 @@ NATIVE_CMAKE_FLAGS+=(-DBC_LOAD_FLAGS="-target ${target} --sysroot=/opt/${target}
 
 cmake -B build-native -S enzyme -GNinja "${NATIVE_CMAKE_FLAGS[@]}"
 
-# Only build blasheaders (and eventually tblgen)
-ninja -C build-native -j ${nproc} blasheaders
+# Only build blasheaders and tblgen
+ninja -C build-native -j ${nproc} blasheaders enzyme-tblgen
 
 # 2. Cross-compile
 CMAKE_FLAGS=()
 CMAKE_FLAGS+=(-DENZYME_EXTERNAL_SHARED_LIB=ON)
 CMAKE_FLAGS+=(-DBC_LOAD_HEADER=`pwd`/build-native/BCLoad/gsl/blas_headers.h)
+CMAKE_FLAGS+=(-DEnzyme_CONFIG_TABLEGEN_EXE=`pwd`/build-native/tools/enzyme-tblgen/enzyme-tblgen)
```

Comment:
Doesn't seem to be using this one?
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=20752&view=logs&j=023fc726-910f-52b9-2466-adec1eae67d0&t=7bf96d55-6405-5d0a-4d10-93a2cdf2f87c&l=433